src/index.ts

```diff
@@ -72,7 +73,13 @@ export { noopCacheProvider, IServiceProviderConfig, SamlConfig };
 const withSpidAuthMiddleware = (
   acs: AssertionConsumerServiceT,
   clientLoginRedirectionUrl: string,
-  clientErrorRedirectionUrl: string
+  clientErrorRedirectionUrl: string,
+  logCallback?: (
```

Comment:
naming: you don't know how this callback will be used by the caller

```suggestion
  callback?: (
```